template/hooks/junkchecker/pre-commit

```diff
@@ -12,9 +12,8 @@ then
 	for FILE in $(git diff-index --cached --name-status $against -- | cut -c 3-); do
 		if [ -f "$FILE" ]
 		then
-			for line in $(cat "$junkchecker_phrases_file")
+			while IFS='' read -r line || [[ -n "$line" ]];
```

Comment:
Code from https://stackoverflow.com/questions/10929453/read-a-file-line-by-line-assigning-the-value-to-a-variable/10929511#10929511